samples/sqltransport/store-and-forward/sample.md

```diff
@@ -0,0 +1,101 @@
+
+
+### Sender project
+
+The Sender does not store any data. It mimics the front-end system where orders are submitted by the users and passed via the bus to the back-end. It is configured to use SQL Server transport with NHibernate persistence.
+
+snippet:SenderConfiguration
+
+The Sender registers two custom behaviors, one for the send pipeline and the other for the receive pipeline.
+
+
+#### Send pipeline
+
+The new behavior is added at the beginning of the send pipeline (in Version 5) or in the routing stage (in Version 6).
+
+snippet:SendThroughLocalQueueBehavior
+
+The behavior ignores:
+- Messages sent from a handler, because the incoming message will be retried ensuring the outgoing messages are eventually delivered. 
+- Deferred messages, because their destination is the local timeout manager satellite. 
+
+The behavior captures the destination of the message in a header and overrides the original value so that the message is actually sent to the local endpoint (put at the end of the endpoint's incoming queue).
+
+NOTICE: In Version 5 other properties of a message (such as defer time) are not captured in this example. In order to use similar feature in production, make sure to capture all relevant information (e.g. defer time).
+
+
+#### Receive pipeline
+
+In the receive pipeline the new behavior is placed just before loading the message handlers (in Version 5) or in the physical processing stage (in Version 6).
+
+snippet:ForwardBehavior
+
+If the message contains the headers used by the send-side behavior, it is forwarded to the ultimate destination instead of being processed locally. This is the first time the remote database of the Receiver endpoint is contacted. Should it be down, the retry mechanism kicks in and ensures the message is eventually dispatched to the destination. In this example the retry mechanism is configured to retry up to 100 times every 10 seconds.
+
+snippet:SlrConfig
+
+NOTE: In case when sender's and receiver's databases cannot be accessed in a distributed transaction, the `ForwardBehavior` has to include a `TransactionScope` that suppresses the ambient transaction before forwarding the message. 
+
+### Receiver project
+
+The Receiver mimics a back-end system. It is set up to use SQLServer transport with NHibernate persistence. The following code configures it to use the [*multi-instance* mode](/nservicebus/sqlserver/deployment-options.md#modes-overview-multi-instance) of the SQL Server transport.
+
+snippet:ReceiverConfiguration
+
+NOTE: Multi-instance mode is deprecated in Version 6 of NServiceBus (Version 3 of SQL Server transport) and will be removed in the next major version. By that time an alternative store-and-forward solution will be provided.
```

Comment:
Here I'd say core has nothing to do with multi-instance. How about:
"Multi-instance mode is supported for backwards compatibility in SQL Server transport Version 3 and will be removed in the next major version." + link http://docs.particular.net/nservicebus/upgrades/sqlserver-2to3.md#sql-server-transport-multi-instance-support.